dbt/include/starrocks/macros/adapters/relation_helpers.sql

```diff
@@ -90,7 +90,15 @@
       {%- for item in distributed_by -%}
         {{ item }} {%- if not loop.last -%}, {%- endif -%}
       {%- endfor -%}
-    ) BUCKETS {{ buckets }}
+    ) 
+    {%- if buckets is not none -%}
+      BUCKETS {{ buckets }}
+    {%- elif adapter.is_before_version("2.5.7") -%}
+      {%- set msg -%}
+        [buckets] must set before version 2.5.7, current version is {{ adapter.current_version() }}
```

Comment:
If the lower version does not support auto bucket, should we set the bucket to 10?

Reply:
I more lean on the side of no default, as to not cause weird edge cases/bugs for users. (Running different versions and/or updating database version)